sdk/src/wallet/operations/transaction/high_level/minting/mint_nfts.rs

```diff
@@ -54,7 +54,12 @@ impl MintNftParams {
 
     /// Set the address and try convert to [`Bech32Address`]
     pub fn try_with_address(mut self, address: impl ConvertTo<Bech32Address>) -> crate::wallet::Result<Self> {
-        self.address = Some(address.convert()?);
+        self.address = Some(
+            address
+                .convert()
+                .map_err(AddressError::from)
```

Comment:
There is a bunch of `.map_err(AddressError::from)` that can be removed in these files